controllers/virtualmachineinstance_controller.go

```diff
@@ -19,48 +19,51 @@ package controllers
 import (
 	"context"
 	"github.com/go-logr/logr"
-	"sigs.k8s.io/controller-runtime/pkg/event"
```

Comment:
lets split the includes to the different groups with line between please

the inner ones
the github / controller runtime maybe together with them (not sure if a line need to be in between them)
ours

Reply:
done